components/TpsBar/TpsBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Text } from 'degen';
+import { LOW_TPS } from '../../constants/vehoney';
+import { useTPS } from '../../hooks/useTPS';
+
+/**
+ * @params None
+ * @description Range slider regarding borrow and repay feature
```

Comment:
I assume you copy pasted the style and forgot to remove this?

Reply:
yes , fixed